### PR TITLE
2020 talks

### PR DESCRIPTION
**Pull request by Greh**
Not completed yet, but I'm making a PR so people can view it

### CODE REVIEW

File: content/talks/adventures-in-babysitting-webscraping-for-pythong-and-html-novices/contents.lr [hunk 1]
```diff
@@ -0,0 +1,14 @@
+_model: talk
```
Comment (cache-rules) [inline content/talks/adventures-in-babysitting-webscraping-for-pythong-and-html-novices/contents.lr]: The file path should be "content/talks/adventures-in-babysitting-webscraping-for-**python**-and-html-novices/contents.lr"

**Comment by nnja** [comment]
@Greh Where are the headshots located?

**Comment by Greh** [comment]
I haven't uploaded the ones we have because we decided to post the speaker bios without them until we get the rest

**Comment by roadsideseb** [comment]
I've restructured the speakers so that it works with more than two speakers. As previously, a talk shows up multiple times in that case so that every speaker gets equal representation. It's the best short-term solution...we might want something different next year.

**Review by cache-rules — APPROVED** [review]

File: content/talks/adventures-in-babysitting-webscraping-for-pythong-and-html-novices/contents.lr (hunk 1 above)
Comment (Greh) [reply content/talks/adventures-in-babysitting-webscraping-for-pythong-and-html-novices/contents.lr]: goddamnit, I do that all time time

**Pull request merged**